### ISSUE TITLE
Invalid URL 'None': No schema supplied. Perhaps you meant http://None?

### ISSUE THREAD

**Issue by EIzquierdo** [samapriya/geeup#27]
I use geeup to upload images to the assets. I didn't use it in long time (around six months ago). I updated to the last version and I got the error when I run geeup upload.
It looks like the package works well because it can create the new imageCollection or recognize if it exists but when it is processing the images, I get always: Invalid URL 'None': No schema supplied. Perhaps you meant http://None?
I'm using python 3.6.9 in ubuntu 18.
Has something changed or am I doing something wrong?
Thanks!

**Comment by samapriya** [comment]
Hi @EIzquierdo 
I am sorry to hear you are having issues. Usually the http:// None error occurs if the tool cannot get to the upload url inside Google Earth Engine. MIght I suggest trying to run the init tool with geeup

```
geeup selsetup
```

Let me know how it goes

**Comment by samapriya** [comment]
I tested the latest version to upload imagery and could not replicate the issue so I am closing this issue for now assuming it might be localized but feel free to reopen if new information becomes available.

**Comment by kkraoj** [comment]
I faced this issue today and these are the steps I took to fix it. TLDR: It was an issue with google authorization.

system configuration: Windows 64 bit, python 3.6, geeup v.0.4.4

I had done `geeup init` and `geeup selsetup` but still encountered the error `Invalid URL 'None': No schema supplied. Perhaps you meant http://None?` while trying `geeup upload <many more args>`  command. The error is because of "None" returned by [this line](https://github.com/samapriya/geeup/blob/dfdfdba668f61a2e1e1fce4fa4d0d68272deb8bc/geeup/batch_uploader.py#L350) in batch_uploader.py. I again tried `geeup upload <many more args>`  but this time kept a close eye on my phone. After entering password for upload, google sent a 2 step request to my phone. Once I confirmed that, uploader worked like a charm. 

If more people face this issue, maybe in a future version @samapriya can add an `if` statement after the referenced line to check whether you have a valid upload URL.

I hope this helps.    
Big thanks to @samapriya for making this awesome API and keeping the repo updated.
